truss/templates/trtllm-briton/src/engine.py

```diff
@@ -52,15 +63,33 @@ def is_port_available(port, host="localhost"):
 
 def briton_monitor(briton_process):
     while True:
-        if briton_process.poll() is not None:
+        if briton_process.exitcode is not None:
             print(
-                f"Briton process has exited with code {briton_process.returncode}, exiting truss server"
+                f"Briton process has exited with code {briton_process.exitcode}, exiting truss server"
             )
             pid = os.getpid()
             os.kill(pid, signal.SIGKILL)
         time.sleep(1)
 
 
+def start_briton(config_pbtxt_path, briton_env, tp_count):
```

Comment:
We run into issues like
```
I0000 00:00:1725882971.094089       7 fork_posix.cc:77] Other threads are currently calling into gRPC, skipping fork() handlers                                                                                           
```
when subprocess.popen used now that we are spinning off other processes for the fsm generation. GRPC does not play well with fork(), so we can set multiprocessing to spawn a new process instead.